app/jobs/warehouse_reports/run_active_veterans_job.rb

```diff
@@ -15,7 +16,7 @@ def perform(params)
 
       user = User.find(params[:current_user_id])
       report.user_id = user.id
-      report.parameters[:visible_projects] = if user.can_view_all_reports?
+      report.parameters[:visible_projects] = if GrdaWarehouse::DataSource.can_see_all_data_sources?(user)
```

Comment:
I think this must have been a mistake to check `can_view_all_reports?` here, becuase we use this later on to render "Unlimited" vs "Limited," and the "Limited" value shows a tooltip saying something like "this report is limited to the following projects that are viewable to the user: x,y,z"
@eanders

Reply:
You are correct, that was a historical use of the permission.  Not sure `can_see_all_data_sources?` is correct, but it is more correct.  It should probably just always use `GrdaWarehouse::Hud::Project.viewable_by(user).pluck(:id, :ProjectName)`.  At some level, there will need to be an awareness that very few people can see all projects and your reports will be limited to the projects you can see.